packages/block-library/src/paragraph/deprecated.js

```diff
@@ -11,7 +11,7 @@ import {
 	RawHTML,
 } from '@wordpress/element';
 import {
-	getColorClassName,
+	getColorClassName, getFontSizeClass,
```

Comment:
Maybe in a new line?

Reply:
Sure, will fix 👍 

PhpStorm adds those imports now, I wasn't aware that it adds in the same line 😅

Reply:
1bab5b3a2ab1b738449cf6f12dd55c7bfc4dc939 ✅